test/e2e/datadogagent_test.go

```diff
@@ -436,7 +436,7 @@ func DeploymentWithClusterAgentEnabled(t *testing.T) {
 	defer ctx.Cleanup()
 
 	name := "foo"
-	agentdeployment := utils.NewDatadogAgent(namespace, name, fmt.Sprintf("datadog/agent:%s", "6.14.0"), &utils.NewDatadogAgentOptions{ClusterAgentEnabled: true})
+	agentdeployment := utils.NewDatadogAgent(namespace, name, fmt.Sprintf("datadog/agent:%s", "7.22.0"), &utils.NewDatadogAgentOptions{ClusterAgentEnabled: true})
```

Comment:
Didn’t you want to put `7.21` everywhere it was `6.14` before ?

Reply:
Yes, it's in the DCA test though, so hopefully should not matter.